### PR TITLE
sfml: update to 2.4.2

### PR DESCRIPTION
**Pull request by qme**

### CODE REVIEW

File: mingw-w64-sfml/PKGBUILD [hunk 1]
```diff
@@ -20,13 +20,14 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
...
+source=("http://www.sfml-dev.org/files/SFML-${pkgver}-sources.zip"
         mingw-w64-msys2.patch)
-sha256sums=('f75096b2dc9cae67e10a28dbbefc9fe02e9dbe2e1ed50f2e208046bae9d3c9a4'
-            '63a429098a12ee5ee2111d2df735ef5dfff693ef5b1b76a8d9a93e2c978408b6')
+sha256sums=('8ba04f6fde6a7b42527d69742c49da2ac529354f71f553409f9f821d618de4b6'
+            'd7cf7f13f76f7e77bc759616d73fc81b97552132da99005b45ef538bcfdcc338')
 noextract=(SFML-${pkgver}-sources.zip)
 
 prepare() {
+  rm -r -f SFML-${pkgver}
```
Comment (mati865) [inline mingw-w64-sfml/PKGBUILD]: It will return error if there was no `SFML-${pkgver}`

Comment (qme) [inline mingw-w64-sfml/PKGBUILD]: With -f option it should not.

Comment (mati865) [inline mingw-w64-sfml/PKGBUILD]: Ok, looks like I forgot what `-f` does for rm 😄

Comment (qme) [inline mingw-w64-sfml/PKGBUILD]: It is not clear to me how the installation of FindSFML.cmake works... in my first commit I considered correct looking for it within ${srcdir}, it worked properly on my MSYS2 environment, but sadly failed on tea integration...

**Comment by hcorion** [comment]
I think a csfml rebuild could also be added to this PR.

**Comment by Alexpux** [comment]
Yes, add ````csfml```` here too.

**Comment by qme** [comment]
CSFML rebuild added.

**Comment by qme** [comment]
CSFML build failed... "FindSFML.cmake" finding looks problematic. Please cosider my previous comment about it too.

**Pull request merged**